README.md

```diff
@@ -2,16 +2,25 @@
 
 > Lagos Digital Summit is a platform for all tech enthusiasts, including business owners, freelancers, start-up founders, MDAs, FinTechs, students within and outside of the state, corporate bodies, and the general public.
 
+> This Website has an Homepage that highlights:
+
+- A header with logo and navigation bar
+- Main Program Section
+- Featured Speakers Section
+- Partners Section
+- Footer.
+
+> This Website has compatible with different screen sizes and very responsive.
+
 
 ## Built With
 
 - Major languages - HTML /CSS and Javascript
 - Frameworks - Bootstrap 5
-- Technologies used
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://samlorlah.github.io/lagos_digital_summit/)
```

Comment:
- [ ] Kindly fix your demo link because it is not working. It gives me a blank page.